python/runtime/cudaq/algorithms/py_draw.cpp

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+#include <pybind11/complex.h>
+#include <pybind11/stl.h>
+
+#include "utils/OpaqueArguments.h"
+#include "cudaq/algorithms/draw.h"
```

Comment:
Whenever I see includes in this order, I immediately think it's a bug. Guess I've been doing LLVM for too long.

At any rate, the LLVM argument of always including local sources first is a good one as it prevents one from writing broken .h files that have dependences on other headers that the file doesn't include itself.

Reply:
Will fix this